Other/MergeShops/SunlightZoneHouseMerge.cs

```diff
@@ -0,0 +1,102 @@
+/*
+name: Sunlight Zone House Merge
+description: This bot will farm the items belonging to the selected mode for the Sunlight Zone House Merge [2289] in /sunlightzone
+tags: sunlight, zone, house, merge, sunlightzone, stern, song, guest, taras, temporary, rest, disgruntled, mi
+*/
+//cs_include Story/AgeOfRuin/CoreAOR.cs
+//cs_include Scripts/Story\ShadowsOfWar\CoreSoW.cs
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreFarms.cs
+//cs_include Scripts/CoreAdvanced.cs
+using Skua.Core.Interfaces;
+using Skua.Core.Models.Items;
+using Skua.Core.Options;
+
+public class SunlightZoneHouseMerge
+{
+    private IScriptInterface Bot => IScriptInterface.Instance;
+    private CoreBots Core => CoreBots.Instance;
+    private CoreFarms Farm = new();
+    private CoreAdvanced Adv = new();
+    private static CoreAdvanced sAdv = new();
+
+    private CoreSoW SoW = new();

```

Comment:
This is not needed, CoreSoW just needs to be in the includes, no need to make an instance of it

Reply:
to piggy back, you only need todo these when your using the third part of it (SoW for instance here) in the script to use a function)